code/modules/mining/mine_items.dm

```diff
@@ -233,8 +233,8 @@
 	drill_verb = "drilling"
 	autodrill = 1
 	drill_sound = 'sound/weapons/drill.ogg'
-	digspeed = 20
-	digspeed_unwielded = 30
+	digspeed = 15
+	digspeed_unwielded = 15
```

Comment:
Wielded/Unwielded should not be the same

Reply:
Only pickaxes have a wield function, so this should not make any difference at all.

Reply:
Future-proof your code.